mttl/models/lightning/expert_module.py

```diff
@@ -24,6 +24,14 @@
 
 
 class LightningTrainingMixin:
+
+    @property
+    def experts_names(self):
+        return self.model.experts_names
+
+    def get_expert_instance(self, name):
```

Comment:
this is only available for multiexpert, i am not sure to add this dependency here